startup.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
 KEEP_ALIVE_TIMEOUT="${KEEP_ALIVE_TIMEOUT:-61000}"
+MAX_HTTP_HEADERS_SIZE="${MAX_HTTP_HEADERS_SIZE:-32768}"
 
-exec npm start -- --keepAliveTimeout $KEEP_ALIVE_TIMEOUT
+export NODE_OPTIONS="--max-http-header-size=$MAX_HTTP_HEADERS_SIZE"
```

Comment:
```suggestion
export NODE_OPTIONS="--max-http-header-size=$MAX_HTTP_HEADERS_SIZE ${NODE_OPTIONS}"
```

I could easily imagine someone's passing this environment variable as part of their infrastructure configuration. It would be very hard to figure out why it does not work as expected because here we override it's value.